package.json

```diff
@@ -1286,25 +1296,16 @@
                 "key": "ctrl+w l",
                 "command": "workbench.action.navigateRight",
                 "when": "!editorTextFocus && !terminalFocus"
-            },
-            {
-                "key": "ctrl+w q",
-                "command": "workbench.action.closeActiveEditor",
-                "when": "!editorTextFocus && !terminalFocus && !filesExplorerFocus && !searchViewletFocus"
-            },
-            {
-                "key": "ctrl+Escape",
-                "command": "workbench.action.focusActiveEditorGroup",
-                "when": "terminalFocus"
             }
         ]
     },
     "scripts": {
         "lint": "eslint --ext .ts src",
         "format": "prettier --ignore-path .gitignore --ignore-path .prettierignore --write .",
+        "keybind": "node ./scripts/keybindings/index.cjs && prettier --write ./package.json",
         "test": "node ./out/test/runTest.js",
         "pretest": "npm run test-compile",
-        "vscode:prepublish": "webpack --mode production",
+        "vscode:prepublish": "npm run keybind && webpack --mode production",
```

Comment:
Could we commit the autogenerated result, so that it's visible in the git tree? That makes things more obvious/explicit.

Reply:
> Or developers need to run keybind manually?

Yes. 

I added it to prepublish for convenience.

Reply:
That needs to be documented somewhere, it's probably time we added a CONTRIBUTING.md.

But ideally it shouldn't be needed at all, instead the generated result should be committed to the repo so that the repo state matches what would be published.

Reply:
I'm  actually in the middle of writing a CONTRIBUTING.md!